hoodie-client/src/test/java/com/uber/hoodie/common/TestRawTripPayload.java

```diff
@@ -41,12 +42,20 @@
     private String rowKey;
     private byte[] jsonDataCompressed;
     private int dataSize;
+    private boolean isDeleted;
 
     public TestRawTripPayload(String jsonData, String rowKey, String partitionPath, String schemaStr) throws IOException {
         this.jsonDataCompressed = compressData(jsonData);
         this.dataSize = jsonData.length();
         this.rowKey = rowKey;
         this.partitionPath = partitionPath;
+        this.isDeleted = false;
+    }
+
+    public TestRawTripPayload(String rowKey, String partitionPath) throws IOException {
+        this.rowKey = rowKey;
+        this.partitionPath = partitionPath;
+        this.isDeleted = true;
```

Comment:
Have isDeleted exposed constructor and have a default constructor which sets it to false. Not a fan of creating a subset of fields in the constructor.